packages/boxed-expression-component/src/selection/BeeTableSelectionContext.tsx

```diff
@@ -801,6 +809,14 @@ export function BeeTableSelectionContextProvider({ children }: React.PropsWithCh
     }
   }, [activeDepth, containerCellCoordinates, depth, dispatch, isSelectionHere, parentActiveCell, selection]);
 
+  function getCellColSpan(cell?: Set<BeeTableCellRef>): number {
+    let colSpan = 1;
+    if (cell) {
+      cell.forEach((e) => (colSpan = e.colSpan ?? 1));
+    }
+    return colSpan;
+  }
```

Comment:
Using `Set` here just because of the structure of `currentRefs`. During debugging I didn't see a case, when `set` would contain amount of cells higher than 1.